src/util/selinux.c

```diff
@@ -43,25 +44,44 @@
  */
 int selinux_file_context(const char *dst_name)
 {
-    security_context_t scontext = NULL;
+    struct selabel_handle *handle = NULL;
+    char *scontext = NULL;
+    int ret;
 
     if (is_selinux_enabled() == 1) {
```

Comment:
This one looks bit strange but this fits man is_selinux_enabled():
"is_selinux_enabled() returns 1 if SELinux is running or 0 if it is not."

The only other place where we using is_selinux_enabled() is:
./src/util/sss_semanage.c:85:    if (!is_selinux_enabled()) {

But I do not think we should try to unify this as current construction fits our official coding guidelines :)